staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1/types_jsonschema.go

```diff
@@ -68,24 +68,24 @@ type JSONSchemaURL string
 // JSONSchemaPropsOrArray represents a value that can either be a JSONSchemaProps
 // or an array of JSONSchemaProps. Mainly here for serialization purposes.
 type JSONSchemaPropsOrArray struct {
-	Schema      *JSONSchemaProps  `protobuf:"bytes,1,opt,name=schema"`
-	JSONSchemas []JSONSchemaProps `protobuf:"bytes,2,rep,name=jSONSchemas"`
+	Schema      *JSONSchemaProps  `json:"schema,omitempty" protobuf:"bytes,1,opt,name=schema"`
+	JSONSchemas []JSONSchemaProps `json:"JSONSchemas,omitempty" protobuf:"bytes,2,rep,name=JSONSchemas"`
```

Comment:
was this really `jSONSchemas` before? I believe the name is not important, but I am not deep into proto. @smarterclayton can you comment?

Reply:
same here, these are already published in the openapi schema like this:

```
   "io.k8s.apiextensions-apiserver.pkg.apis.apiextensions.v1beta1.JSONSchemaPropsOrArray": {
    "description": "JSONSchemaPropsOrArray represents a value that can either be a JSONSchemaProps or an array of JSONSchemaProps. Mainly here for serialization purposes.",
    "required": [
     "Schema",
     "JSONSchemas"
    ],
```

and are persisted to etcd that way as well. match the existing published schema and open an issue to track fixing this in v1

Reply:
And as expected, jsoniter serialized (to etcd) using the upper-case name as well. So to avoid breaking decoding, we have to keep these wrong field names :sigh: